screens/request_profile.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, FlatList} from 'react-native';
+
+const window = Dimensions.get("window")
+
+let colors = ['#6DC4E0', '#605DF1']
+
+let items = [
+  {
+    name: 'Item',
+    q: 'Q',
+    comment: 'Comments',
+    check: require('./../assets/images/check.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    q: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  }
+]
+
+export default class Request extends React.Component { 
+  render() {
+    return (
+    <View>
+          <View style={styles.container}>
+            <View style={styles.topBanner} />
+            <Image style={styles.shoppingBagicon}
+                source={require('./../assets/images/shopping_bag.png')} />
+            <Image style={styles.personIcon}
+                source={require('./../assets/images/person.png')} />
+            <Image style={styles.arrowIcon}
+                source={require('./../assets/images/back_arrow.png')} />
+            <Text style={styles.browseText}>Browse Other Options</Text>
+            <View style={{justifyContent: 'center', alignItems: 'center'}}>
+              <View style={styles.whiteCard} />
+              <View style={styles.nameBox} />
+              <Text style={styles.nameText}>Bob - CVS</Text>
+           </View>
+           </View>
+                <View style={{top: 315, justifyContent: 'center', alignItems: 'center'}}>
+                  <FlatList
+                  data={items}
+                  keyExtractor={(item, index) => index.toString()}
+                  renderItem={
+                    ({item, index}) => {
+                    return (
+                        <View>
```

Comment:
There are too many `<View>` components in this render function. You should just one `<View>` component since it's just a rectangle that contains content.

Also, make sure to nest component inside one another. This will make using `flexboxes` easier.